### PR TITLE
ucloud 0.1.3

### PR DESCRIPTION
**Pull request by lixiaojun629**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/ucloud.rb [hunk 1]
```diff
@@ -1,8 +1,17 @@
 class Ucloud < Formula
   desc "The official tool to managment your ucloud services"
   homepage "https://www.ucloud.cn"
-  url "https://ucloud-sdk.dl.ufileos.com/ucloud-cli-0.1.2.tar.gz"
-  sha256 "f41aaa6bf8063b3ccd0c5e41d3dc90f46f9fb45066513da8e306456f1f332071"
+  url "https://github.com/ucloud/ucloud-cli.git",
+      :tag => "0.1.3",
+      :revision => "907f6391705400ec5cb59353e38834cf0193db3b"
+
+  bottle do
```
Comment (fxcoudert) [inline Formula/ucloud.rb]: Don't touch (and don't add) bottle blocks please

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/ucloud.rb (hunk 1 above)
Comment (lixiaojun629) [reply Formula/ucloud.rb]: Sorry, It is a mistake when execute 'git rebase -i upstream/master'.  fixed.

File: Formula/ucloud.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 class Ucloud < Formula
   desc "The official tool to managment your ucloud services"
   homepage "https://www.ucloud.cn"
-  url "https://ucloud-sdk.dl.ufileos.com/ucloud-cli-0.1.2.tar.gz"
-  sha256 "f41aaa6bf8063b3ccd0c5e41d3dc90f46f9fb45066513da8e306456f1f332071"
+  url "https://github.com/ucloud/ucloud-cli.git",
+      :tag => "0.1.3",
+      :revision => "907f6391705400ec5cb59353e38834cf0193db3b"
```
Comment (fxcoudert) [inline Formula/ucloud.rb]: Can we continue to build from tarball please?

Comment (lixiaojun629) [reply Formula/ucloud.rb]: Building from github makes it easier to win the trust of users. 
If there are other reasons, I can change it back.  Thanks.

**Comment by lixiaojun629** [comment]
I was just wondering if there is anything else to be done on my end, or it's good to go. Thanks.

**Comment by lixiaojun629** [comment]
@fxcoudert could we please promote or merge this pr?

**Comment by fxcoudert** [comment]
- Please build from the github tarball: 
https://github.com/ucloud/ucloud-cli/archive/0.1.3.tar.gz
- Version 0.1.3 is marked as “pre-release” on github, which is not acceptable for Homebrew. We only ship stable versions.

**Comment by lixiaojun629** [comment]
> * Version 0.1.3 is marked as “pre-release” on github, which is not acceptable for Homebrew. We only ship stable versions.

Version 0.1.3 has passed the test and  is marked as a stable version now. Thank you.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**